pyproject.toml

```diff
@@ -61,6 +61,8 @@ distro = "^1.6.0"
 rich = {extras = ["jupyter"], version = "^11.0.0"}
 httplib2 = "<0.20,>=0.19.1"
 pyparsing = "<3,>=2.4.0"
+wandb = "^0.12.12"
```

Comment:
Hey @soumik12345 , we dont usually add dependencies in our core dependency project.toml. I would ask you to remove changes to the pyproject.toml and put them instead in `src/zenml/ntegrations/wandb/__init__.py`. We have another way of controlling these dependencies